src/components/form/input.js

```diff
@@ -1,10 +1,85 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
-const Input = ({ text }) => <input>{text}</input>
+const InputWrapper = styled.div`
+    position: relative;
+    width: 100%;
+`
+
+const StyledInput = styled.input`
+    background: none;
+    color: var(--color-black);
+    font-size: 1.6rem;
```

Comment:
we could consider adding a css variable for this since we are using this size everywhere also